homeassistant/components/streamlabswater/sensor.py

```diff
@@ -17,6 +26,38 @@
 NAME_YEARLY_USAGE = "Yearly Water"
 
 
+@dataclass(frozen=True, kw_only=True)
+class StreamlabsWaterSensorEntityDescription(SensorEntityDescription):
+    """Streamlabs sensor entity description."""
+
+    value_fn: Callable[[StreamlabsData], StateType]
+
+
+SENSORS: tuple[StreamlabsWaterSensorEntityDescription, ...] = (
+    StreamlabsWaterSensorEntityDescription(
+        key="daily_usage",
+        name=NAME_DAILY_USAGE,
```

Comment:
Would we not use the translation key?

Reply:
Wanted to do that in a follow up as I thought adding device and has entity name was already out of the scope of this PR :P

Reply:
I think it's not out of scope, please do it.